ceph/ceph_admin/daemon.py

```diff
@@ -0,0 +1,42 @@
+"""Cephadm orchestration host operations."""
+import logging
+
+from .add import AddMixin
+from .common import config_dict_to_string
+from .orch import Orch
+
+logger = logging.getLogger(__name__)
+
+
+class Daemon(AddMixin, Orch):
+    def op(self, op, config):
+        """
+        Execute the command ceph orch <start|stop|restart|reconfigure|redeploy> <service>.
+        Args:
+            config: command and service are passed from the test case.
+            op: operation parameters ex: restart|start|stop|reconfigure|redeploy
+            Example:
```

Comment:
Minor comment, can this be added before `Args` for clear representation.

Reply:
@sunilkumarn417 we already have  this **Execute the command ceph orch <start|stop|restart|reconfigure|redeploy> <service>**  should i add anything else?

Reply:
sorry, can example part be outside Args or before.